methcomp/blandaltman.py

```diff
@@ -11,33 +12,18 @@ class _BlandAltman(object):
     """Internal class for drawing a Bland-Altman plot"""
 
     def __init__(self, method1, method2,
```

Comment:
If returning instances of this class from the `blandaltman()` the class should no longer be "internal".

Reply:
thats true, so removed the internal (_)